lua/lvim/core/lualine/components.lua

```diff
@@ -164,7 +173,17 @@ return {
     function()
       local current_line = vim.fn.line "."
       local total_lines = vim.fn.line "$"
-      local chars = { "__", "▁▁", "▂▂", "▃▃", "▄▄", "▅▅", "▆▆", "▇▇", "██" }
+      local chars = {
+        lvim.icons.progress.Vertical1,
+        lvim.icons.progress.Vertical2,
+        lvim.icons.progress.Vertical3,
+        lvim.icons.progress.Vertical4,
+        lvim.icons.progress.Vertical5,
+        lvim.icons.progress.Vertical6,
+        lvim.icons.progress.Vertical8,
+        lvim.icons.progress.Vertical8,
+        lvim.icons.progress.Vertical9,
+      }
```

Comment:
there's no need to make these configurable, we're probably removing this component anyway since it's not used.